src/api/schema/metrics/mod.rs

```diff
@@ -69,17 +77,54 @@ impl MetricsSubscription {
         })
     }
 
+    /// Component events processed metrics. Streams new data as the metric increases
+    async fn component_events_processed_total(
+        &self,
+        #[arg(default = 1000, validator(IntRange(min = "100", max = "60_000")))] interval: i32,
```

Comment:
After upgrading to async-graphql `v2.0.15`, errors will be reported correctly here.

Reply:
That's great, thanks! This actually caught a silent error, where a `--refresh-interval` below 100 (ms) was previously allowed, but should have failed. Great work on input validation too, btw... really nice to be able to define custom errors so easily!